mappings/net/minecraft/block/Block.mapping

```diff
@@ -155,8 +155,6 @@ CLASS net/minecraft/class_197 net/minecraft/block/Block
 	FIELD field_478 boundingBoxMaxZ D
 		COMMENT Stores the Maximum value on the Z axis for the voxel shape
 	FIELD field_479 sound Lnet/minecraft/class_197$class_200;
-		COMMENT Stores the current block's sound type
-	FIELD field_479 soundGroup Lnet/minecraft/class_200;
```

Comment:
Why deleting it?

Reply:
Duplicated fields aren't a problem anymore, in this case they don't have the same type descriptor that's why there are 2 entry for it.

Reply:
It wasn't building when i ran gradlew build so i thought it wasn't supposed to be there, my bad